src/main/java/net/atomique/ksar/xml/ColumnConfig.java

```diff
@@ -5,29 +5,33 @@
 
 package net.atomique.ksar.xml;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import java.awt.Color;
 
 public class ColumnConfig {
 
+  private static final Logger log = LoggerFactory.getLogger(ColumnConfig.class);
+
   public ColumnConfig(String s) {
     data_title = s;
   }
 
-  public String getData_colorstr() {
-    return data_colorstr;
-  }
+  public void setData_color(String data_color_string) {
+
+    String[] color_indices = data_color_string.split(",");
 
-  public void setData_color(String data_color) {
-    String[] color_indices = data_color.split(",");
-    this.data_colorstr = data_color;
     if (color_indices.length == 3) {
       try {
-        Integer red = new Integer(color_indices[0]);
-        Integer green = new Integer(color_indices[1]);
-        Integer blue = new Integer(color_indices[2]);
-        this.data_color = new Color(red, green, blue);
-      } catch (NumberFormatException ee) {
+        this.data_color = new Color(Integer.parseInt(color_indices[0]),
+                Integer.parseInt(color_indices[1]),
+                Integer.parseInt(color_indices[2]));
+      } catch (IllegalArgumentException iae) {
+        log.error("Column color error for {} - <{}>", data_title, data_color_string, iae);
```

Comment:
This would probably be `warn` as well.

Usually `warn` is printed for recoverable errors (e.g. when default value could be used), and `error` goes for unrecoverable ones (e.g. when no further operation is possible).